### ISSUE TITLE
Nothing happens when I click the script names in the script view

### ISSUE THREAD

**Issue by poof86** [ericadamski/atom-package-script-runner#3]
I'm using Atom on an iMac

**Comment by ericadamski** [comment]
Thanks for submitting an issue!

I would love to get this fixed for you, do you think it is possible for me to glace at your package.json file so I can try and recreate the issue?

**Comment by ericadamski** [comment]
thank you so much for posting this! I will use it and take a look at what the issue is :)

**Comment by ericadamski** [comment]
This works well for me! There was a small issue with your package.json file that might have led the plugin to not parse it correctly.

```js
  "scripts": {
    /* "test": "echo "Error: no test specified" && exit 1", */
    "test": "echo \"Error: no test specified\" && exit 1",
    "watch": "webpack --watch",
  }
```

The other weird thing about Webpack is that they write all of their progress info to `stderr` which I do not display in the little terminal in the plugin.

![kapture 2018-06-25 at 7 39 26](https://user-images.githubusercontent.com/6516758/41848277-ee0dad04-784a-11e8-8562-f932386b2f3e.gif)

I would love to get to the bottom of this error for you! If you ever have time for us to work more closely to see if I can't make this package better for everyone!

**Comment by ericadamski** [comment]
Here is a really quick example of it running (just tested it now to make sure) with a create-react-app start script 😄 

![kapture 2018-06-25 at 7 43 48](https://user-images.githubusercontent.com/6516758/41848489-9ca3fca6-784b-11e8-8a64-bbd86e9d6c50.gif)

**Comment by poof86** [comment]
Your addon still doesn't work for me, I think it's because I changed the Package Manager the default "yarn" to "npm"?

**Comment by ericadamski** [comment]
Hum, I am really sorry it still isn't working 😥

Let me check again with npm and see what happens! I suspect you've found a bug! 🐛

**Comment by ericadamski** [comment]
Hello @poof86! Your issues with using `npm` should be fixed in the newest version [1.0.0](https://github.com/ericadamski/atom-package-script-runner/releases/tag/v1.0.0)[*](https://atom.io/packages/package-script-runner)

let me know if you have any more issues!

Thank you again for taking the time to hep me fix this! ❤️

**Comment by poof86** [comment]
I can confirm that it's working now, thank you for fixing it 🤗 
You squashed that bug 🐛 💦
